index.js

```diff
@@ -159,28 +162,33 @@ app.get('/auth/google', (req, res, next) => {
 //   request.  If authentication fails, the user will be redirected back to the
 //   login page.  Otherwise, the primary route function function will be called,
 //   which, in this example, will redirect the user to the home page.
-app.get('/auth/google/callback', 
+app.get('/auth/google/callback',
   passport.authenticate('google', { failureRedirect: '/login' }),
   function(req, res) {
     const redirect = req.session.returnTo;
     delete req.session.returnTo;
 
-    if (req.session.from_extension) {
```

Comment:
This parameter name change `from_extension` to `from_extension_url` might break the app if the server update and extension update doesn't happen at the same time – which is hard to ensure, given that the extension update is in the browser's control.  

Can you edit the patch to exclude these param change alone?

Reply:
Since the server handles both cases, would it be hard to ensure the server is updated first? 

I think you could deploy the server first, which would still be compatible with the current version of the extension, and then you can publish the extension on the web store.

What do you think?

Reply:
Ah, makes sense. I didn't notice you've preserved the old handling as well. That should do 👍